instrumentation/dubbo/src/test/java/brave/dubbo/ITTracingFilter_Consumer.java

```diff
@@ -48,7 +48,7 @@
   @Before public void setup() {
     server.start();
 
-    String url = "dubbo://" + server.ip() + ":" + server.port() + "?scope=remote&generic=bean";
```

Comment:
I wonder the reason for removing this query string.

Reply:
It was used incorrectly, but did not take effect before 2.7.11，if  used ```generic=bean``` then the parameter type of ```GenericService#$invoke``` args[0] is ```JavaBeanDescriptor```

```java
public class ITTracingFilter_Provider extends ITTracingFilter {
  @Before public void setup() {
    server.service.setFilter("tracing");
    server.service.setGeneric("true");
    server.service.setInterface(GreeterService.class);
    server.service.setRef((method, parameterTypes, args) -> {
      //args[0] instanceof JavaBeanDescriptor
      String arg = (String) args[0];   // test fail
      if (arg.equals("bad")) throw new IllegalArgumentException("bad");
      return currentTraceContext.get() != null
          ? currentTraceContext.get().traceIdString()
          : "";
    });
```

see 
```GenericImplFilter#invoke```